opal/mca/common/ucx/common_ucx_wpool.h

```diff
@@ -298,9 +304,6 @@ static inline int _periodical_flush_nb(opal_common_ucx_wpmem_t *mem, opal_common
 {
     int rc = OPAL_SUCCESS;
 
-    winfo->inflight_ops[target]++;
-    winfo->global_inflight_ops++;
```

Comment:
in case you are wondering, why this was removed - I moved them outside - to the calls which actually need to increase inflight ops, if they are in here, I noticed that it will increase inflights ops after an win_unlock, and it will cause an assertion failure

Reply:
We can discuss it, but, the flush call should not increment operations, imo. It's a flush, and all it needs to do is to flush when the condition is met. If you post an operation, you should increment inflight ops in case they are outstanding.